common/app/common/TrailsToRss.scala

```diff
@@ -86,7 +86,7 @@ object TrailsToRss extends implicits.Collections {
         val imageMetadata = new Metadata()
         trailAsset.caption.foreach({ d => imageMetadata.setDescription(stripInvalidXMLCharacters(d)) })
         trailAsset.credit.foreach { creditName =>
-          val credit = new Credit(null, null, creditName)
+          val credit = new Credit(null, null, stripInvalidXMLCharacters(creditName))
```

Comment:
I wonder if we should have a more generic approach that strip invalid chars from all fields rather than doing it one field at a time when it crashes.